locales/en/translation.json

```diff
@@ -123,24 +120,66 @@
   "We couldn’t find any products. Try again later?": "We couldn’t find any products. Try again later?",
   "Who can access this shared link?": "Who can access this shared link?",
   "You are in offline mode. We are trying to reconnect, but you may need to": "You are in offline mode. We are trying to reconnect, but you may need to",
+  "a_or_b": "<0></0> or <2></2>",
   "anErrorOccurred": "An error occurred. Try the <1>home page</1>?",
-  "and": "and",
   "completed": "completed",
   "credentialsHoldTime": "The credentials to your Salesforce org will only be held for <2>{{token_minutes}}</2> minute or until your requested installation is complete.",
   "credentialsHoldTime_plural": "The credentials to your Salesforce org will only be held for <2>{{token_minutes}}</2> minutes or until your requested installation is complete.",
+  "credentialsHoldTime_0": "The credentials to your Salesforce org will only be held for <2>{{token_minutes}}</2> minutes or until your requested installation is complete.",
+  "credentialsHoldTime_1": "The credentials to your Salesforce org will only be held for <2>{{token_minutes}}</2> minutes or until your requested installation is complete.",
+  "credentialsHoldTime_2": "The credentials to your Salesforce org will only be held for <2>{{token_minutes}}</2> minutes or until your requested installation is complete.",
+  "credentialsHoldTime_3": "The credentials to your Salesforce org will only be held for <2>{{token_minutes}}</2> minutes or until your requested installation is complete.",
+  "credentialsHoldTime_4": "The credentials to your Salesforce org will only be held for <2>{{token_minutes}}</2> minutes or until your requested installation is complete.",
+  "credentialsHoldTime_5": "The credentials to your Salesforce org will only be held for <2>{{token_minutes}}</2> minutes or until your requested installation is complete.",
```

Comment:
Ah: https://www.i18next.com/translation-function/plurals#languages-with-multiple-plurals 👍

Reply:
@jgerigmeyer Yep. And it's easiest for our translators if they just have to fill in translations for the right plurals for their language, rather than adding new keys.